ui/admin/app/controllers/scopes/scope/scopes.js

```diff
@@ -12,7 +12,7 @@ export default class ScopesScopeScopesController extends Controller {
   /**
    * @type {string}
    */
-  @computed('model.currentScope.{isGlobal,isOrg}')
+  @computed('intl', 'model.currentScope.{isGlobal,isOrg}')
```

Comment:
Still trying to understand ember but can you explain what this change is for? Is it so the getter can also use `intl`?

Reply:
Did eslint complain about not declaring `intl` as a dependent key?

Reply:
@ZedLi @randallmorey 

This error comes with the update of `eslint-plugin-ember` and the update of the recommended rules.

As Randy mentioned, this change is due to eslint complaining about dependencies to be declared statically in computed properties. [More info about this rul in here](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/require-computed-property-dependencies.md).

Happy to discuss other approaches :smiley: